frontend/src/components/GameList/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+import * as api from 'lib/api';
+
+class GameList extends Component {
+  state = {
+    games: [],
+    isLoading: false,
+  };
+
+  componentDidMount() {
+    this.loadGames();
+  }
+
+  loadGames = () => {
+    this.setState({ isLoading: true });
+    api.getGames().then((games) => this.setState({ games, isLoading: false }));
+  };
+
+  handleAddGame = () => {
+    const { games } = this.state;
+    this.setState({ isLoading: true });
+    api.createGame({ name: `Game ${games.length + 1}` }).then(this.loadGames);
+  };
+
+  render() {
+    const { games, isLoading } = this.state;
+
+    if (isLoading) {
+      return <p>Loading...</p>;
+    }
```

Comment:
Is there value to replacing the entire UI with a loading state?

Not sure the user needs to know, and they'll have access to the rest of the games in the list.

It might only be useful for the _initial_ load?

Reply:
Happy to make this a separate ticket. Wouldn't block this PR by any means.

Reply:
Yep, this is only for initial load and arguably doesn't even need to be there, just made it as a crutch for local dev. Happy to 🗑️ too